velox/dwio/parquet/tests/writer/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+add_executable(velox_parquet_writer_sink_test SinkTests.cpp)
+
+add_test(
+  NAME velox_parquet_writer_sink_test
```

Comment:
To be consistent as velox_dwio_parquet_reader_test and more general, why don't we make it velox_parquet_writer_test

Reply:
Fixed, by using `velox_parquet_writer_test`.

Reply:
I meant either we change the class `SinkTest` to `ParquetWriterTest` together with the test name `velox_parquet_writer_sink_test `

But I'm fine if we want to keep the class name `SinkTest`, then let's change the test name back to `velox_parquet_writer_sink_test`

Thanks!

Reply:
Got it, done.